cluster-service/Makefile

```diff
@@ -57,6 +57,7 @@ deploy:
 	  --set shard.cxSecretsKeyVaultMiClientId="${CX_SECRETS_KV_MI_CLIENT_ID}" \
 	  --set shard.maestroRestUrl="http://maestro.maestro.svc.cluster.local:8000" \
 	  --set shard.maestroGrpUrl="maestro-grpc.maestro.svc.cluster.local:8090" \
+	  --set shard.Id=$$(python3 -c "import uuid; print(uuid.uuid5(uuid.NAMESPACE_DNS, '${MGMT_AKS_NAME}'))") \
```

Comment:
I don't think the mgmt aks name matches the uuid predefined namespace dns scheme.

We can have our own by using our own UUID. Let's use 916f9976-e1c0-4afd-b84c-5d5c94fbeaed as the namespace UUID and let's store it in a variable that we call for example PROVISION_SHARDS_IDS_NAMESPACE_UUID

Reply:
Let's also add doc to it:

	// PROVISION_SHARDS_IDS_NAMESPACE_UUIDis the UUID used as the Namespace ID
	// for generating Azure Provision Shard IDs. The value was generated
	// using an UUIDv4 with the github.com/google/uuid library.
	// *WARNING* Do not change this value as changing it changes how the
	// provision shard ids are generated and consumers of the generated names
	// rely on predictability of the generated names based on the given input.

Reply:
// PROVISION_SHARDS_IDS_NAMESPACE_UUID is the UUID used as the Namespace ID
// for generating Azure Provision Shard IDs. The value was generated
// using an UUIDv4 with the github.com/google/uuid library.
Why can't we use uuid.NAMESPACE_DNS, as the namespace ID? Its defined as part of the uuidv5 rfc and would not change.

Reply:
Because the MGMT_AKS_NAME is conceptually not a uuid.NAMESPACE_DNS. It doesn't fit in that category nor any of the other provided ones so it is more appropriate to define our own.